### DISCUSSION TITLE
Chance to contribute code?

### DISCUSSION THREAD

**Discussion by mf-guse** [MarcGiffing/bucket4j-spring-boot-starter#344] [General]
It seems to be a little bit quiet around development in this project.

I'd love to use bucket4j-spring-boot-starter in one project but I need to enhance the functionality a little bit. Is there any chance to contribute code which is accepted timely?

**Comment by MarcGiffing** [comment]
Any contributions are welcome, but can you describe the enhancements a little bit?

**Comment by mf-guse** [comment]
First contribution is a fixed redis-jedis example.

We would like to use the annotation-based ratelimit configuration. In our application we figured out that in this case no metrics are written. In the current implementation the RateLimitAspect does not set MetricHandlers.

I would like to improve this.

What is the desired procedure? I create a fork and then do a pull request?

**Comment by mf-guse** [comment]
I created a pull request containing my changes to get redis-jedis example running with @RateLimiting.
